config/jobs/kubernetes/sig-cluster-lifecycle/kubespray.yaml

```diff
@@ -0,0 +1,28 @@
+# kubeadm release and master tests
+periodics:
+  kubernetes-incubator/kubespray:
+  - name: ci-kubernetes-e2e-kubeadm-gce-master
+    interval: 2h
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: quay.io/kubespray/kubespray:v2.7
+        args:
+        - --repo=k8s.io/kubernetes=master
+        - --repo=github.com/kubernetes-incubator/kubespray=master
+        - --timeout=320
+        - --upload=gs://kubernetes-jenkins/logs
+        - --scenario=kubernetes_e2e
+        - --
+        - --cluster=
+        - --deployment=kubespray
+        - --kubeadm=ci
+        - --provider=kubespray
```

Comment:
there was a bit of a problem recently with.
note that this value will end up being handled as a skeleton / nil provider as it is currently unknown.

if we want to set the value to anything else it has to be "skeleton"!